.gitlab-ci.yml

```diff
@@ -121,7 +121,7 @@ check:
   <<:                              *docker-env
   <<:                              *test-refs
   script:                          &check-script
-    - SKIP_WASM_BUILD=1 time cargo check --locked --verbose --workspace --features runtime-benchmarks
```

Comment:
Had to remove `--features runtime-benchmarks` because none of the remaining crates expose such a feature anymore.